### ISSUE TITLE
Stuck in Reboot

### ISSUE THREAD

**Issue by spencerjason** [Floris497/mac-pixel-clock-patch-V2#250]
After all the great success everyone was having, I thought that I could patch my Mac Mini 2012 for High Sierra finally. 

I ran the patcher, rebooted, and now I'm stuck in a reboot loop (the progress bar barely even moves).

I can boot into safe mode, but I can't browse anything to unpatch from Safe Mode. Is a restore from TM backup (or reinstall OS) my only option here?

**Comment by Floris497** [comment]
Can you boot using (cmd + v) and see where booting gets stuck?

**Comment by spencerjason** [comment]
I didn't even think about that. I got impatient (and needed to boot ASAP) so I pulled a TM backup from the 10.13.2 update, which was less than 30 minutes before I patched.

**Comment by Floris497** [comment]
Ah so because the kernel cache is being rebuilt and some other processes.. it can take a while for first boot. Doesn't mean something is wrong.

**Comment by spencerjason** [comment]
No. I got through the 10.13.2 update (which did take about an hour). When I patched, I received a few warnings, but they didn't seem related directly to the patch so I didn't think anything of it. For example, one was related to the iTunes Helper, another the Adobe Helper. I had taken a screenshot to document, but when I restored from TM that obviously vanished. I didn't think to store it on a different drive.

But the reboot after the patch caused it to startup, show the Apple logo and status bar, but only move a tiny fraction before rebooting again and doing the same in a loop.

**Comment by Floris497** [comment]
Ah well.. those show up because of "update_dyld_shared_cache" I should supress that output when the script is running.. maybe

**Comment by spencerjason** [comment]
Apparantely I've been running in sleep for quite some time and this issue came back. That allowed me to rule out Pixel Clock patch as the culprit, but I still was stuck in a kernel panic reboot that was so fast, verbose and safe were also affected. 

On a whim to save recent data on my desktop and downloads, I booted the mac mini disk into target mode and connected to my laptop. I thought I'd try booting it as an external drive there and see what would happen. Same kernel panic, however this time the reboot took me back to my laptop OS instead of the target drive. 

Oddly, it still gave me the developers message and asked if I wanted to send the report to apple. I looked at the details, and it was Lilu kext causing the issue. I went into the library, deleted the associated extensions, and now Pixel Clock is working perfect at 2560x1080 60hz again.

So...  THANK YOU. :D
